src/mame/igs/igs_m027.cpp

```diff
@@ -217,20 +306,22 @@ void igs_m027_state::lhdmg_xor_map(address_map &map)
 	igs_mahjong_xor_map(map);
 
 	map(0x4000000c, 0x4000000f).r(FUNC(igs_m027_state::lhdmg_unk2_r));
+	map(0x40000018, 0x4000001b).umask32(0x000000ff).w(FUNC(igs_m027_state::jking02_misc_w));
```

Comment:
This definitely isn’t right – bit 7 of PPI port C has something to do with sound banking, but it isn’t as simple as just banking the whole sample ROM.  There are very obvious issues with sounds during gameplay.  Unfortunately these games don’t have a sound test.

Reply:
yeah, as I've said above, some of the other games point at it being a 128kb/128kb bank, so it could just be that fruitpar and oceanpar are using it in avery basic way, banking complete 256kb segments, and the others are not, or the logic differs between games (which would fit with everything else IGS loved to change for the sake of it)